### DISCUSSION TITLE
微信小号扫码登录失败，并一直重复尝试，uu有遇到过这个问题吗

### DISCUSSION THREAD

**Discussion by PengfeiMiaoTW** [meteorOSS/WeChatBc#20] [Q&A] [answered]
<img width="1385" alt="Screenshot 2024-02-27 at 18 34 36" src="https://github.com/meteorOSS/wechat-bc/assets/115978506/ba2910da-a0d4-4de7-bc1d-b9673713e905">

另一个常用的微信账号登录是正常的，这个失败的账号是能正常登录的没有异常

我的猜想是由于账号不常用，在用WeChatClient模拟登录的时候会有额外的安全验证？

**Comment by PengfeiMiaoTW** [comment]
<img width="1159" alt="Screenshot 2024-02-28 at 17 00 01" src="https://github.com/meteorOSS/wechat-bc/assets/115978506/a5414675-2ce2-4e08-95a9-55df799f949a">
错误码 1100

**Comment by meteorOSS** [comment, marked as answer]
已经修复了该问题，更新即可

**Reply by PengfeiMiaoTW** [reply]
没问题了，强！
